OrbitCore/CrashHandler.h

```diff
@@ -4,9 +4,21 @@
 #pragma once
 
 #include <string>
+#include <atomic>
+
+#ifdef _WIN32
+#include <client/crashpad_client.h>
+#endif
 
 class CrashHandler {
  public:
-  CrashHandler();
-  static void SendMiniDump();
+  CrashHandler(const std::wstring& dump_path);
```

Comment:
You almost always want to make constructors `explicit` in C++, especially the ones with just one argument. Otherwise somebody might write this:
```
CrashHandler handler = someStringVariable;
```
That looks like an assignment but is actually a construction of a CrashHandler object.